modules/enabling-intel-gaudi-ai-accelerators.adoc

```diff
@@ -1,10 +1,10 @@
 :_module-type: PROCEDURE
 
-[id='enabling-habana-gaudi-devices_{context}']
-= Enabling Habana Gaudi devices
+[id='enabling-intel-gaudi-ai-accelerators_{context}']
+= Enabling Intel Gaudi AI Accelerators
 
 [role='_abstract']
-Before you can use Habana Gaudi devices in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator. 
+Before you can use Intel Gaudi AI Accelerators in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator. 
```

Comment:
Same comment again, but this time for the HabanaAI Operator. I'm assuming that this name has not changed either.

Reply:
Correct, no change to the Operator naming currently.